library/ziti.c

```diff
@@ -1613,6 +1613,15 @@ int ziti_context_run(ziti_context ztx, uv_loop_t *loop) {
     return ZITI_OK;
 }
 
+int ziti_refresh(ziti_context ztx) {
+    if (!ztx->enabled) return ZITI_DISABLED;
+
+    ZTX_LOG(INFO, "application requested service refresh");
```

Comment:
feels like a DEBUG to me or even maybe removing it. i called it -- i could log it?